src/accel/SetupOptions.hh

```diff
@@ -26,45 +26,67 @@ struct AlongStepFactoryInput;
 /*!
  * Control options for initializing Celeritas SD callbacks.
  *
- * These affect only the \c HitManager construction that is responsible for
- * reconstructing CPU hits and sending directly to the Geant4 detectors.
+ * By default, Celeritas connects to Geant4 sensitive detectors so that it
+ * reconstructs full-fidelity hits with all available step information.
+ * - If your problem has no SDs, you must set \c enabled to \c false.
+ * - By default, steps that do not deposit energy do not generate any hits.
+ * - To improve performance and memory usage, determine what quantities (time,
+ *   position, direction, touchable, ...) are required by your setup's
+ *   sensitive detectors and set all other attributes to \c false.
+ * - Reconstructing the full geometry status using \c locate_touchable is the
+ *   most expensive detector option. Disable it unless your SDs require (e.g.)
+ *   the volume's copy number to locate a detector submodule.
+ * - Some reconstructed track attributes (such as post-step material) are
+ *   currently never set because they are rarely used in practice. Contact the
+ *   Celeritas team or submit a pull request to add this functionality.
  *
  * Various attributes on the step, track, and pre/post step points may be
  * available depending on the selected options.
- * - Disabling \c track will leave \c G4Step::GetTrack as \c nullptr
- * - Enabling \c locate_touchable will also set \c Material and \c
- *   MaterialCutsCouple
- * - Enabling \c track will set particle the \c Charge attribute on the
- *   pre-step
+ * - Disabling \c track will leave \c G4Step::GetTrack as \c nullptr .
+ * - Enabling \c track will set the \c Charge attribute on the
+ *   pre-step.
  * - Requested post-step data including \c GlobalTime, \c Position, \c
  *   KineticEnergy, and \c MomentumDirection will be copied to the \c Track
- *   when the combination of options is enabled
- * - Track and Parent IDs will \em never be a valid value since Celeritas track
- *   counters are independent from Geant4 track counters.
- * - Some within-step properties (material, material cuts couple, and
+ *   when the combination of options is enabled.
+ * - Some pre-step properties (\c Material and \c MaterialCutsCouple, and
  *   sensitive detector) are always updated. Post-step values for those are not
  *   set.
+ * - Track and Parent IDs will \em never be a valid value since Celeritas track
```

Comment:
```suggestion
 * - Track and Parent IDs will \em never be a valid value since Celeritas track
```
humm .. sounds like they are random value and thus not even needed to kept?  Do we mean just that the value there are internal to Celeritas and completely de-corelated from any similar sounding Geant4 values?

Reply:
The track/parent IDs in the reconstructed track aren't valid since we don't set them. We don't set them because yes they're not directly correlated to Geant4 tracks.